src/components/BookItem.js

```diff
@@ -8,7 +9,7 @@ const BookItem = (props) => {
   const removeBookFromStore = () => {
     dispatch(removeBookFromApi(id));
   };
-
+  const percentage = 65;
   return (
     <div className="bookItemContainer">
       <div className="booksSection">
```

Comment:
- [ ] You have done well getting to this point and your design is great 👍. However, there are some designs that need to match Zeplin's design as it is a requirement to match Zeplin's design.

- [ ] Let's make the `comments`, `remove`, and `edit` to have a right border as it is in the Zeplin design, please let's add it to make sure our design matches that of Zeplin.

- [ ] The circular progress bar looks a little bit different from the design. Let's make it look like a real progress bar by making the radius clear.

- [ ] Let's make the inputs match that of Zeplin's by increasing the width as seen in Zeplin's design.

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/q4NEVQW.png) | ![](https://i.imgur.com/KYuF3Hf.png)